setup/itopdesignformat.class.inc.php

```diff
@@ -17,7 +17,14 @@
  * You should have received a copy of the GNU Affero General Public License
  */
 
-define('ITOP_DESIGN_LATEST_VERSION', '1.7'); // iTop >= 2.7.0
+
+/**
+ * iTop Datamodel XML format version
+ * And also iTop core version
```

Comment:
```suggestion
 * And also iTop core version (since 2020-10-16)
```

Should we add this to remember or not?

Reply:
Not sure... Besides it's not that clear to me when reading the meeting minutes which decision was made ? 
I've added a subject for the dev team meeting, we may add this information afterwards ?